package/sdbus-cpp/Config.in.host

```diff
@@ -0,0 +1,11 @@
+config BR2_PACKAGE_HOST_SDBUS_CPP
+	bool "sdbus-c++-xml2cpp"
+	select BR2_PACKAGE_HOST_PKGCONF
```

Comment:
you don't need this - just add in the "host-pkgconf" to the Dependencies - it get's build from there. You do selects when you want those tools but nothing depends on them for like a script or something like that.